dietpi/func/dietpi-banner

```diff
@@ -237,6 +238,15 @@ $GREEN_LINE"
 		(( ${aENABLED[1]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[1]} $GREEN_SEPARATOR $(uptime -p 2>&1)"
 		# CPU temp
 		(( ${aENABLED[2]} == 1 )) && echo -e "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[2]} $GREEN_SEPARATOR $(print_full_info=1 G_OBTAIN_CPU_TEMP 2>&1)"
+		# RAM usage
+		if (( ${aENABLED[17]} == 1 ))
+		then
+			ramused="$(free -h --si | mawk 'NR==2 {printf $3}' 2>&1)"
+			ramtotal="$(free -h --si | mawk 'NR==2 {printf $2}' 2>&1)"
+			ramperc="$(free -b | mawk 'NR==2 {printf "%.0f", 100*$3/$2}' 2>&1)"
```

Comment:
I guess we can enhance performance by getting the whole string with a single `free` and `mawk` call ... although difficult if we want to allow different units. I suggest to use MiB:
```suggestion
			ramusage=$(free -b | mawk 'NR==2 {CONVFMT="%.0f"; print $3/1024^2" MiB of "$2/1024^2" MiB ("$3/$2*100"%)"}')
```
```console
root@DietPi:/boot/dietpi# free -b | mawk 'NR==2 {CONVFMT="%.0f"; print $3/1024^2" MiB of "$2/1024^2" MiB ("$3/$2*100"%)"}'
311 MiB of 1967 MiB (16%)
```
What do you think?

Reply:
It sounds definitely better 😄 

Ive also been checkin the `dietpi-config` script, and I saw that there is already something similar to display the memory (in the `Menu_PerformanceOptions` function). 

What do you think of extracting the display of memory logic into `dietpi-globals`?

Reply:
@MichaIng Ive made the changes that you have requested... For now I will just calculate the ram usage in the banner file, in the future when I have more time, I will try to create a `G_OBTAIN_RAM_USAGE` so that it can be used globally

Reply:
Jep that is fine.